databroker/experimental/config.yml

```diff
@@ -0,0 +1,8 @@
+authentication:
+  allow_anonymous_access: true
+trees:
+  - path: /
+    tree: "server_ext:MongoAdapter.from_uri" # "server_ext:MongoAdapter.from_uri"
```

Comment:
This will only work if the current working directory is `databroker/experimental`. Use the full module path `databroker.experimental.server_ext` so that it works from anywhere.